Makefile

```diff
@@ -50,30 +50,49 @@ endif
 endif
 export BASEIMAGE
 
+ifeq (${IMAGE_REPO}, )
+  IMAGE_REPO = "openebs"
+  export IMAGE_REPO
+endif
+
+# Specify the docker arg for repository url
+ifeq (${DBUILD_REPO_URL}, )
+  DBUILD_REPO_URL="https://github.com/openebs/node-disk-manager"
+  export DBUILD_REPO_URL
+endif
+
+# Specify the docker arg for website url
+ifeq (${DBUILD_SITE_URL}, )
+  DBUILD_SITE_URL="https://openebs.io"
+  export DBUILD_SITE_URL
+endif
+
+export DBUILD_ARGS=--build-arg DBUILD_DATE=${DBUILD_DATE} --build-arg DBUILD_REPO_URL=${DBUILD_REPO_URL} --build-arg DBUILD_SITE_URL=${DBUILD_SITE_URL} --build-arg ARCH=${ARCH}
```

Comment:
if `DBUILD_DATE` env is not set by user, it should get the date from the build system. Otherwise that label will be empty
```
"Labels": {
                "org.label-schema.build-date": "",
                "org.label-schema.description": "OpenEBS Node Disk Manager",
                "org.label-schema.name": "node-disk-manager",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.url": "https://openebs.io",
                "org.label-schema.vcs-url": "https://github.com/openebs/node-disk-manager"
            }
```